dspace-server-webapp/src/main/java/org/dspace/app/rest/DiscoveryRestController.java

```diff
@@ -202,18 +203,16 @@ public RepresentationModel getFacetValues(@PathVariable("name") String facetName
         dsoTypes = emptyIfNull(dsoTypes);
 
         if (log.isTraceEnabled()) {
-            log.trace("Facetting on facet " + facetName + " with scope: " + StringUtils.trimToEmpty(dsoScope)
-                          + ", dsoTypes: " + String.join(", ", dsoTypes)
-                          + ", prefix: " + StringUtils.trimToEmpty(prefix)
-                          + ", query: " + StringUtils.trimToEmpty(query)
-                          + ", filters: " + Objects.toString(searchFilters)
-                          + ", page: " + Objects.toString(page));
+            log.info("Facetting on facet {} with scope: {}, dsoTypes: {}, prefix: {}, query: {}, filters: {}, page: {}", facetName, StringUtils.trimToEmpty(dsoScope), String.join(", ", dsoTypes), StringUtils.trimToEmpty(prefix), StringUtils.trimToEmpty(query), Objects.toString(searchFilters), Objects.toString(page));
         }
 
+        log.info("Facetting on facet {} with scope: {}, dsoTypes: {}, prefix: {}, query: {}, filters: {}, page: {}", facetName, StringUtils.trimToEmpty(dsoScope), String.join(", ", dsoTypes), StringUtils.trimToEmpty(prefix), StringUtils.trimToEmpty(query), Objects.toString(searchFilters), Objects.toString(page));
         try {
             FacetResultsRest facetResultsRest = discoveryRestRepository
                 .getFacetObjects(facetName, prefix, query, dsoTypes, dsoScope, configuration, searchFilters, page);
-
+            String[] canDisplayLabel = DSpaceServicesFactory.getInstance().getConfigurationService()
+                    .getArrayProperty("discovery.facet.hasLabels");
+            log.info("canDisplayLabel: " + Arrays.toString(canDisplayLabel));
```

Comment:
This `log.info` looks to be for debugging... it should be removed as it's not useful in production logs.